src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working.");
```

Comment:
The stop message should also be more informative and consistent with the start message. Consider using a similar pattern, such as 'Bulldozer has stopped working.'